bc_obps/compliance/tests/service/bc_carbon_registry/test_apply_compliance_units_service.py

```diff
@@ -163,6 +163,8 @@ def test_get_apply_compliance_units_page_data_success(
             trading_name="Test Corp",
         )
 
+        mock_bccr_service.validate_holding_account_ownership.return_value = True
```

Comment:
It would be a good test to assert that `validate_holding_account_ownership` is called after applying units.